### ISSUE TITLE
Query.returning() wrong alias implementation 

### ISSUE THREAD

**Issue by dmkulazhenko** [coleifer/peewee#2046]
Hi there!

I tried to use peewee as SQLGenerator (yeah, I know that it's very strange and ... know, I just have a reason of doing this shit :trollface:)

That's OK:
```
c = User.insert(username='Petr').returning(User.username)
c.sql()
[stdout]: ('INSERT INTO "user" ("username", "password", "last_activity") VALUES (%s, %s, %s) RETURNING "user"."username"', ['Petr', 'password', 1571594114])
```
But if I will try to use alias in returning section:
```
c = User.insert(username='Petr').returning(User.username.alias('u'))
c.sql()
[stdout]: ('INSERT INTO "user" ("username", "password", "last_activity") VALUES (%s, %s, %s) RETURNING "u"', ['Petr', 'password', 1571594114])
```
That isn't OK, because I want something like:
```
('INSERT INTO "user" ("username", "password", "last_activity") VALUES (%s, %s, %s) RETURNING "user"."username" AS "u"', ['Petr', 'password', 1571594114])
```

idk, if that problem reproduces with other queries (not insert)...

Maybe, I just cannot find method, that do this staff, btw I can't find anything about this [here](http://docs.peewee-orm.com/en/latest/peewee/querying.html#returning-clause).

**Comment by dmkulazhenko** [comment]
peewee==3.11.2
psycopg2-binary==2.8.4
(PostgreSQL)

**Comment by dmkulazhenko** [comment]
@coleifer, any reply?)

**Comment by coleifer** [comment]
Fixed in c2dc7ca7caaafc0840b771de6e5dfc5cbabe83a0

**Comment by dmkulazhenko** [comment]
@coleifer, thx, have a nice day!
